backend/src/prompts/templates/validator.j2

```diff
@@ -2,7 +2,7 @@ You are an expert validator. You can help with validating the answers to the tas
 
 Your entire purpose is to return a "true" or "false" value to indicate if the answer has fulfilled the task, along with a reasoning to explain your decision.
 
-You will be passed a task and an answer. You need to determine if the answer is correct or not.
+You will be passed a task and an answer. You need to determine if the answer is correct or not, ensuring that the task's specific requirements are addressed.

```

Comment:
Why are there changes to the validator template if we are disabling it for the webAgent? Again promptfoo tests should be added for these changes.

Reply:
I disabled the second validation, I will add promptfoo tests.

Reply:
I see, thanks for explaining